orkaudio/audiocaptureplugins/voip/VoIpSession.cpp

```diff
@@ -4325,6 +4325,16 @@ void VoIpSessions::ClearLocalPartyMap()
        m_localPartyMap.clear();
 }
 
+std::map<unsigned long long, VoIpSessionRef> VoIpSessions::getByIpAndPort()
+{
+       return m_byIpAndPort;
+}
+
+LoggerPtr VoIpSessions::getLogger()
+{
+       return m_log;
+}
+
```

Comment:
why does logger needs to exposed outside?

Reply:
removed